### ISSUE TITLE
Importing geoviews breaks `show_grid=True` for matplotlib backend in holoviews

### ISSUE THREAD

**Issue by zertrin** [pyviz/geoviews#307]
Importing geoviews breaks usage of `show_grid=True` for matplotlib backend in holoviews _even when not using geoviews at all_ with the following error:

```
AttributeError: 'AxesSubplot' object has no attribute 'gridlines'
```

See minimal working example here for the full stacktrace: https://anaconda.org/zertrin/2019-04-03-holoview-reproduce-show_grid-bug/notebook?version=2019.04.03.1541

Commenting out `import geoviews as gv` restarting and re-running the notebook works. Thus it seems related to Geoviews and I post the issue here. If it is a Holoview bug, this issue can be moved over there instead.

**Comment by ahuang11** [comment]
It seems to only error with overlays; this works
```
(hv.Curve([0, 1, 2]) + hv.Curve([4, 5, 6])).opts('Curve', show_grid=True)
```
![image](https://user-images.githubusercontent.com/15331990/55602842-5e0b2400-571c-11e9-8554-f9aded75a996.png)

but not `(hv.Curve([0, 1, 2]) * hv.Curve([4, 5, 6])).opts(show_grid=True)`

**Comment by philippjfr** [comment]
I believe this was fixed in #308
